WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -0,0 +1,323 @@
+                       image: Constants.avatarPlaceholder)
+        setupIconImage(siteIconImageView,
+                       containerView: siteIconContainerView,
+                       image: Constants.siteIconPlaceholder)
+        setupSiteTitle()
+        setupPostDate()
+        setupSiteStackView()
+
+        setupPostTitle()
+        setupPostSummary()
+        setupFeaturedImage()
+        setupPostCounts()
+
+        setupControlButtons()
+        setupControlsStackView()
+
+        setupSeparatorView()
+    }
+
+    func setupContentView() {
+        contentView.backgroundColor = .listForeground
+    }
+
+    func setupContentStackView() {
+        contentStackView.translatesAutoresizingMaskIntoConstraints = false
+        contentStackView.axis = .vertical
+        contentStackView.alignment = .leading
+        contentStackView.spacing = Constants.ContentStackView.spacing
+        contentView.addSubview(contentStackView)
+    }
+
+    func setupIconImage(_ imageView: UIImageView, containerView: UIView, image: UIImage?) {
+        containerView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.layer.cornerRadius = Constants.iconImageSize / 2.0
+        imageView.layer.masksToBounds = true
+        imageView.image = image
+        containerView.addSubview(imageView)
+        siteStackView.addArrangedSubview(containerView)
+    }
+
+    func setupSiteTitle() {
+        siteTitleLabel.translatesAutoresizingMaskIntoConstraints = false
+        siteTitleLabel.font = .preferredFont(forTextStyle: .subheadline).semibold()
+        siteTitleLabel.numberOfLines = 1
+        siteTitleLabel.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+        siteTitleLabel.setText("Site Title") // TODO: Remove
+        siteStackView.addArrangedSubview(siteTitleLabel)
+    }
+
+    func setupPostDate() {
+        postDateLabel.translatesAutoresizingMaskIntoConstraints = false
+        postDateLabel.font = .preferredFont(forTextStyle: .subheadline)
+        postDateLabel.numberOfLines = 1
+        postDateLabel.textColor = .secondaryLabel
+        postDateLabel.setText("Post Date") // TODO: Remove
+        siteStackView.addArrangedSubview(postDateLabel)
+    }
+
+    func setupSiteStackView() {
+        siteStackView.translatesAutoresizingMaskIntoConstraints = false
+        siteStackView.setCustomSpacing(Constants.SiteStackView.avatarSpacing, after: avatarContainerView)
+        siteStackView.setCustomSpacing(Constants.SiteStackView.iconSpacing, after: siteIconContainerView)
+        siteStackView.setCustomSpacing(Constants.SiteStackView.siteTitleSpacing, after: siteTitleLabel)
+        contentStackView.addArrangedSubview(siteStackView)
+    }
+
+    func setupPostTitle() {
+        postTitleLabel.translatesAutoresizingMaskIntoConstraints = false
+        postTitleLabel.font = .preferredFont(forTextStyle: .title3).semibold()
```

Comment:
In the Figma design, this is marked as `bold`, but the generated code and (I believe) the display actually uses `semibold`. 🤔 

I think going with `semibold` is correct here, but I'll let @osullivanchris confirm! 🙂

Reply:
Yeah it seems from the guidelines that Apple are using semibold up to and including Title3, so semibold would be correct here.